cloud-pak-installer/templates/roks-workspace-configuration.json

```diff
@@ -1,9 +1,7 @@
 {
```

Comment:
It looks like you accidentally renamed the cp4d3.0 template over the roks template.  Something is wrong here.